helm/fiftyone-teams-app/templates/_helpers.tpl

```diff
@@ -299,10 +311,13 @@ Create a merged list of environment variables for fiftyone-teams-cas
     secretKeyRef:
       name: {{ $secretName }}
       key: fiftyoneAuthSecret
-- name: NEXTAUTH_BASEPATH
-  value: "/cas/api/auth"
 - name: NEXTAUTH_URL
   value: {{ printf "https://%s/cas/api/auth" .Values.teamsAppSettings.dnsName | quote }}
+- name: HUB_MONGODB_URI
```

Comment:
Just saw that we're setting CAS_DATABASE_NAME over in the `values.yaml`, so that makes me feel better - but I also saw that `HUB_DATABASE_NAME` is set to `hub`... which now makes me wonder, what is the `hub` database?  Is this a third database?